src/assets/scripts/sections/product.js

```diff
@@ -79,6 +83,34 @@ export default register('product', {
         this.updateImages.bind(this),
       );
     }
+
+    this.initImageSwitch();
+  },
+
+  initImageSwitch() {
+    if (!$(selectors.productThumbs).length) {
+      return;
+    }
+
+    $(selectors.productThumbs)
+      .on('click', (evt) => {
+        evt.preventDefault();
+
+        const imageId = evt.currentTarget.dataset.thumbnailId;
+        this.switchImage(imageId);
+        this.setActiveThumbnail(imageId);
+      })
+      .on('keyup', this.handleImageFocus.bind(this));
+  },
+
+  handleImageFocus(evt) {
+    if (evt.keyCode !== keyboardKeys.ENTER) {
+      return;
+    }
+
+    $(`${selectors.productFeaturedImage}`, this.$container)
```

Comment:
No need for string interpolation: `$(selectors.productFeaturedImage, this.$container)`.